tests/test_e2e_10_mef_eline.py

```diff
@@ -1369,3 +1370,588 @@ def test_155_removing_evc_metadata_persistent(self):
+        any_flow = self.get_flow_by_vlan_match(data, "4096/4096")
+        assert any_flow["match"] == expected[0]["match"]
+        assert any_flow["priority"] == expected[0]["priority"]
+        assert any_flow["instructions"][0]["actions"] == expected[0]["actions"]
+
+        ra_vlan = randrange(1, 4096)
+        h11, h12 = self.net.net.get('h11', 'h12')
+        h11.cmd('ip link add link %s name ra_vlan type vlan id %s' % (h11.intfNames()[0], ra_vlan))
+        h11.cmd('ip link set up ra_vlan')
+        h11.cmd('ip addr add 10.1.1.11/24 dev ra_vlan')
+        h12.cmd('ip link add link %s name ra_vlan type vlan id %s' % (h12.intfNames()[0], ra_vlan))
+        h12.cmd('ip link set up ra_vlan')
+        h12.cmd('ip addr add 10.1.1.12/24 dev ra_vlan')
+        result = h11.cmd('ping -c1 10.1.1.12')
+        assert ', 0% packet loss,' in result
+
+        # clean up
+        h11.cmd('ip link del ra_vlan')
+        h12.cmd('ip link del ra_vlan')
+        self.net.restart_kytos_clean()
+
+    def test_190_create_untagged_intra_evc(self):
+        """Test create an intra-switch EVC with untagged as uni.tag.value"""
+        payload = {
+            "name": "my evc1",
+            "enabled": True,
+            "uni_a": {
+                "tag": {"tag_type": 1, "value": "untagged"},
+                "interface_id": "00:00:00:00:00:00:00:01:1",
+            },
+            "uni_z": {
+                "tag": {"tag_type": 1, "value": "untagged"},
+                "interface_id": "00:00:00:00:00:00:00:01:2",
+            }
+        }
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        response = requests.post(api_url, json=payload)
+        assert response.status_code == 201, response.text
+        data = response.json()
+        assert 'circuit_id' in data
+        time.sleep(10)
+
+        expected = {
+            "match": {"in_port": 1, "dl_vlan": 0},
+            "actions": [{"action_type": "output", "port": 2}],
+            "priority": 20000
+        }
+
+        api_url = KYTOS_API + '/flow_manager/v2/flows/00:00:00:00:00:00:00:01'
+        response = requests.get(api_url)
+        data = response.json()["00:00:00:00:00:00:00:01"]["flows"]
+        untagged_flow = self.get_flow_by_vlan_match(data, 0)
+        assert untagged_flow["match"] == expected["match"]
+        assert untagged_flow["priority"] == expected["priority"]
+        assert untagged_flow["instructions"][0]["actions"] == expected["actions"]
+
+        h11, h12 = self.net.net.get('h11', 'h12')
+        h11.cmd('ip addr add 100.1.1.11/24 dev %s' % (h11.intfNames()[0]))
+        h12.cmd('ip addr add 100.1.1.12/24 dev %s' % (h12.intfNames()[0]))
+        result = h11.cmd('ping -c1 100.1.1.12')
+        assert ', 0% packet loss,' in result
+
+        # clean up
+        h11.cmd('ip addr del 100.1.1.11/24 dev %s' % (h11.intfNames()[0]))
+        h12.cmd('ip addr del 100.1.1.12/24 dev %s' % (h12.intfNames()[0]))
+        self.net.restart_kytos_clean()
```

Comment:
```suggestion
```

Reply:
Done in commit [89d4f76e090d3feb6275dfb07494de4b37869739](https://github.com/kytos-ng/kytos-end-to-end-tests/pull/213/commits/89d4f76e090d3feb6275dfb07494de4b37869739)